shared/src/business/entities/cases/Case.ts

```diff
@@ -221,20 +221,24 @@ export class Case extends JoiValidationEntity {
    */
   static sortByDocketNumber(cases) {
     return cases.sort((a, b) => {
-      const aSplit = a.docketNumber.split('-');
-      const bSplit = b.docketNumber.split('-');
-
-      if (aSplit[1] !== bSplit[1]) {
-        // compare years if they aren't the same;
-        // compare as strings, because they *might* have suffix
-        return aSplit[1].localeCompare(bSplit[1]);
-      } else {
-        // compare index if years are the same, compare as integers
-        return +aSplit[0] - +bSplit[0];
-      }
+      return Case.docketNumberSort(a.docketNumber, b.docketNumber);
     });
   }
 
+  static docketNumberSort(docketNumberA, docketNumberB) {
+    const aSplit = docketNumberA.split('-');
+    const bSplit = docketNumberB.split('-');
+
+    if (aSplit[1] !== bSplit[1]) {
+      // compare years if they aren't the same;
+      // compare as strings, because they *might* have suffix
+      return aSplit[1].localeCompare(bSplit[1]);
+    } else {
+      // compare index if years are the same, compare as integers
+      return +aSplit[0] - +bSplit[0];
```

Comment:
Curious about the `+` operator use on this line.  Does this implicitly cast the two operands as numbers, or is there another purpose.

Reply:
We didn't add this line, only moved it, so I had to look this one up. It does seem to be only for converting the operand to a number. 

til: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

I'm personally unsure if there's any other context surrounding its use.